v3/plugins/redhat/vscode-camelk/0.0.10/meta.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-camelk
+version: 0.0.10
+type: VS Code extension
+displayName: Tooling for Apache Camel K
+title: Tooling for Apache Camel K
+description: Support for Apache Camel K functionality
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/camel-tooling/vscode-camelk
+category: Language
+firstPublicationDate: '2019-07-10'
+spec:
+  containers:
+  - image: "eclipse/che-remote-plugin-camelk-0.0.10:next"
+    name: vscode-camelk
+    memoryLimit: "1G"
```

Comment:
Minor nitpick: since OpenShift seems to format according to 2^x sizes, I'd prefer this to be `1Gi`

Reply:
created https://github.com/eclipse/che/issues/15326 to handle it as there is the same issue for already existing image. I prefer to handle it separately.